assets/src/edit-story/output/test/getUsedAmpExtensions.js

```diff
@@ -42,7 +42,7 @@ describe('getUsedAmpExtensions', () => {
     );
   });
 
-  it('should include the amp-video script if there are videos', () => {
+  it('should include the amp-video script if there is a video', () => {
```

Comment:
Could we perhaps add a test for `output/story` so that it would include various elements to verify it those produce valid AMP? Looks like currently the tests for story (and page) output don't include elements, maybe some issues would come out already there, for creating fewer bugs?

Reply:
I wanted to, but we currently don't produce valid AMP due to masking.

See discussion at https://github.com/google/web-stories-wp/pull/500#issuecomment-596745077

I'll create a new issue for that discussion/issue and add a note for such a test case.

Reply:
Hmm, does it create a mask (clip-path) even if the mask is set to none?

Reply:
Looking at `getDefaultElementMask()`, yes.

Reply:
Hmph. Okay then, I guess `None` in the select field is just for trolling ;) 

Let's look into this separately then, thanks for confirming.

Reply:
Created #536